ldes-server-compaction/pom.xml

```diff
@@ -101,6 +101,10 @@
             <groupId>org.junit.platform</groupId>
             <artifactId>junit-platform-suite</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.pyroscope</groupId>
+            <artifactId>agent</artifactId>
+        </dependency>
```

Comment:
needed for compaction too ??

Reply:
I'll remove these in ingest as well. they where for the custom tags but running the server locally gives problems with them so I removed them